app/controllers/stories_controller.rb

```diff
@@ -126,7 +126,26 @@ def handle_base_index
     @stories = ArticleDecorator.decorate_collection(@stories)
     set_surrogate_key_header "main_app_home_page"
     response.headers["Surrogate-Control"] = "max-age=600, stale-while-revalidate=30, stale-if-error=86400"
-    render template: "articles/index"
+
+    respond_to do |format|
+      format.html do
+        render template: "articles/index"
+      end
+      format.json do
+        render json: @stories.to_json(
+          only: %i[
+            title path id user_id comments_count positive_reactions_count organization_id
```

Comment:
If this is going to let us reconstruct the home feed we probably need the `main_image` included as well here eh?

Reply:
Ah, good catch! I took this from the json transformation that's currently happening in https://github.com/thepracticaldev/dev.to/blob/38622317b7639b9240bfb7865bd0f8d12841e777/app/views/stories/_main_stories_feed.html.erb#L6, but that doesn't handle the featured story (and thus no main image).